packages/mattermost-components/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"compilerOptions": {
+		"target": "esnext",
+		"module": "esnext",
+		"moduleResolution": "node",
+		"jsx": "react",
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"strict": true,
+		"skipLibCheck": true,
+		"strictNullChecks": true,
+		"noEmit": true,
+		"declaration": true,
+		"outDir": "dist",
+		"paths": {
+			"mattermost-redux/*": ["./node_modules/mattermost-redux/src/*"],
+			"@mattermost/types/*": ["./node_modules/@mattermost/types/src/*"]
```

Comment:
Should this reference the types package directly like how the client package does so TS can build dependencies automatically? I'm not sure how if rollup uses TS or if this is just for IDEs like it is in the main web app package

Reply:
I have no idea.

Reply:
Actually, it looks like since rollup uses babel to handle TS, then this is just needed for the IDE support. This should be fine as-is then